For The Ember Times No. 115 - Writeup on Ember Elevator Pitch (#272)
ember-learn/ember-blog: source/2019-09-13-the-ember-times-issue-115.md

<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
Meet `@model` for route templates 🆕,
Share your thoughts on Apple Music being an Ember app 🍎,

> A designer's & developer's story transitioning to Ember ⚛️➡️🐹,

Reviewer: ```suggestion
A designer's & developer's story of transitioning to Ember ⚛️➡️🐹,
```

[Share your thoughts](https://discuss.emberjs.com/t/apple-music-beta-is-an-ember-app/17026) on having more high profile examples of Ember applications!

---

## [What Makes the Transition to Ember Worthwhile: A New Community Member's Perspective 🐹](https://twitter.com/lenoraporter_/status/1170145563549061122)

In a recent [tweet](https://twitter.com/lenoraporter_/status/1170145563549061122), Senior Product Designer [Lenora Porter (@lenoraporter_)](https://twitter.com/lenoraporter_)

> shared a [video](https://www.instagram.com/tv/B2CCM60AOZG/?igshid=12ivaiobj1y8h) pitching the **most interesting learnings** about Ember for those transitioning from other tech stacks.

Reviewer: ```suggestion
shared a [video](https://www.instagram.com/tv/B2CCM60AOZG/?igshid=12ivaiobj1y8h) to pitch the **most interesting learnings** about Ember to those transitioning from other tech stacks.
```

> After developing apps using Ember with previous experience in React, she talks about learning resources that are useful for those starting out and what makes the transition worthwhile. Among benefits such as Convention over Configuration and integrated tooling, she also highlights how a supportive community makes learning Ember very straightforward.

Reviewer: Jessica, while I understood what you mean to say, I think a more clear description may be:
```suggestion
Lenora, who has prior experience in React, talks about useful learning resources for those starting out Ember and what makes the transition worthwhile. In addition to Convention over Configuration and integrated tooling, she highlights how a supportive community makes learning Ember very straightforward.
```
